test/connectors/ERSConnectorSpec.scala

```diff
@@ -98,43 +97,21 @@ class ERSConnectorSpec extends UnitSpec with MockitoSugar with OneAppPerSuite wi
         Future.successful(HttpResponse(INTERNAL_SERVER_ERROR))
       )
 
-      val result = await(ersConnectorUnderTest.validateFileData(callbackData, schemeInfo))
-      result.status shouldBe INTERNAL_SERVER_ERROR
-    }
-
+      }
+//TODO hmm
+      "file-validator returns 5xx" in {
```

Comment:
tests usually test things

Reply:
Yes, I am a fool who missed this I have wrote them now